src/exercises/day-3/safe-ffi-wrapper.rs

```diff
@@ -16,7 +16,7 @@
 mod ffi {
     use std::os::raw::{c_char, c_int};
     #[cfg(not(target_os = "macos"))]
-    use std::os::raw::{c_long, c_ulong, c_ushort};
+    use std::os::raw::{c_long, c_ulong, c_ushort, c_uchar};
```

Comment:
I notice that `uchar` was introduced in Rust 1.64. We don't have a strict lower bound for the Rust compiler version in the course — but there is often someone who tries to use the one shipped in our gLinux Debian install. I should probably just clean up the installation instructions to be clear that people need to install from rustup.rs.

Reply:
It might be worth mentioning "MSRV" in those instructions -- it's a good concept to be aware of anyway.